config/services.yaml

```diff
@@ -3,6 +3,10 @@ services:
         public: true
         autowire: true
 
+    _instanceof:
+        Rector\PHPStan\Reflection\TypeToCallReflectionResolver\TypeToCallReflectionResolverInterface:
+            tags: ['rector.type_to_call_reflection_resolver']
+
```

Comment:
Instanceof is buggy in Symfony PHAR, we can't use it.
Luckily, we don't need tags. Just use autowired arrays